src/Web3/Types.elm

```diff
@@ -218,3 +218,32 @@ type CallType
     | Async
     | Setter
     | Getter
+
+
+{-| Available ethereum denominations.
+-}
+type EthDenomination
+    = Wei --        1
+    | Kwei --       1000
```

Comment:
I'm wondering if we can drop the decimal unit prefixes since they are easily computed from any base unit. i.e. `1000 wei == 1 Kwei == 1 Ada`